apps/omg_watcher_info/lib/omg_watcher_info/db/block.ex

```diff
@@ -92,6 +92,56 @@ defmodule OMG.WatcherInfo.DB.Block do
     |> Paginator.set_data(paginator)
   end
 
+  @spec query_last_24_hour(Ecto.Query.t()) :: Ecto.Query.t()
+  def query_last_24_hour(query) do
+    now = DateTime.to_unix(DateTime.utc_now())
+    twenty_four_hours = 86_400
+    from(block in query, where: block.timestamp >= ^(now - twenty_four_hours))
+  end
+
+  @doc """
+  Returns the total number of blocks in the last 24 hours
+  """
+  @spec get_count_last_24_hour() :: non_neg_integer()
+  def get_count_last_24_hour do
```

Comment:
```suggestion
  def count_by_timestamp_between(start_datetime, end_datetime) do
```

Same reason as my suggestion to `query_last_24_hour`

Reply:
Done 8946c62 but opting for `count_all_between_timestamps`